tensornetwork/backends/tensorflow/tensorflow_backend.py

```diff
@@ -430,3 +430,26 @@ def eps(self, dtype: Type[np.number]) -> float:
       float: Machine epsilon.
     """
     return tf.experimental.numpy.finfo(dtype).eps
+
+  def pinv(self, tensor: Tensor, rcond: float = 1E-15, hermitian: bool = False) -> Tensor:
```

Comment:
I think you can remove the `hermitian` argument

Reply:
I had included it as it was an argument for numpy. I'll remove it in the next commit. 
Thanks for the review!